### PR TITLE
LPS-94715 | Modernize StorageFormatter utility

### PR DESCRIPTION
**Pull request by mateomustapic**
Hi @wincent ,

Please review modified code :)

### CODE REVIEW

**Comment by wincent** [comment]
ci:test:sf

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/storage_formatter.js [hunk 1]
```diff
@@ -59,47 +55,22 @@ AUI.add(
...
-					suffix: instance.get('addSpaceBeforeSuffix')
-						? STR_SPACE + suffix
-						: suffix
+				var instance = this,
+					addSpaceBeforeSuffix = instance.get('addSpaceBeforeSuffix'),
+					decimalSeparator = instance.get('decimalSeparator'),
+					denominator = instance.get('denominator'),
+					suffixGB = instance.get('suffixGB'),
+					suffixKB = instance.get('suffixKB'),
+					suffixMB = instance.get('suffixMB');
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/storage_formatter.js]: I think this should be one statement per line.

`git grep '^\s*var .+,$'` shows 415 instances of this pattern, but only 4 of them look to be our code and the rest are third-party.

`git grep '^\s*var .+;$'`, on the other hand, looks to return well over 18,000 hits, most of which look like they aren't third-party.

**Comment by wincent** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/storage_formatter.js (hunk 1 above)
Comment (mateomustapic) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/storage_formatter.js]: I agree. I was in doubt to write these statements in this pattern  or one statement per line and I eventually decided to do it this way.

**Comment by wincent** [comment]
Forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/75154

For some reason, GitHub kept erroring out and wouldn't let me make a PR with the branch on your remote, @mateomustapic, so I pushed the exact same commit to my own fork and it let me create it. ¯\\_(ツ)_/¯

**Pull request closed without merging**